src/main/java/Duke.java

```diff
@@ -1,10 +1,189 @@
+import java.util.*;
+
 public class Duke {
+
+    static void level1() {
```

Comment:
Perhaps having a method name that briefly describes what it does would be clearer.